local_run

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Alliander N. V.
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+SCRIPT_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+    pwd -P
+)
+
+images=()
+for dockerfile in $SCRIPT_PATH/dockerfiles/*.Dockerfile; do
+    images+=("$(basename "${dockerfile%.*}")")
+done
+[[ -z $images ]] && echo "No Dockerfiles found" && return 1
+echo "Select image #: "
+select image in "${images[@]}" ; do
+    [[ $image ]] && break
+done
+
+if [ -z "$(docker images -q rcdt\/$image 2> /dev/null)" ]; then
+
+    cache_options=(true false)
+    echo "Build with cache?"
+    select use_cache in "${cache_options[@]}" ; do
+        [[ $use_cache ]] && break
+    done
+    use_cache=$(! "$use_cache" && echo "--no-cache" || echo "")
+
+    sequential_options=(parallel sequential)
+    echo "Colcon build style: "
+    select colcon_build_sequential in "${sequential_options[@]}" ; do
+        [[ $colcon_build_sequential ]] && break
+    done
+    COLCON_BUILD_SEQUENTIAL=$([ "$colcon_build_sequential" == "sequential" ] && echo "true" || echo "false")
+
+    docker build \
+        $use_cache \
+        --build-arg COLCON_BUILD_SEQUENTIAL="$COLCON_BUILD_SEQUENTIAL" \
+        -t "rcdt/$image:local" \
+        -f "$SCRIPT_PATH/dockerfiles/$image.Dockerfile" \
+        "$SCRIPT_PATH/dockerfiles"
+fi
+echo Running $image:local ...
+export IMAGE_TAG="local"
```

Comment:
We kunnen deze eerder definieren en dan gebruiken tijdens het bouwen van de image en de echo direct hierboven